cdap-docs/reference-manual/source/http-restful-api/namespace.rst

```diff
@@ -233,16 +233,16 @@ of ``scheduler.queue.name``.
 
    * - Configuration Name
      - Description
-   * - ``scheduler.queue.name``
-     - Scheduler queue for CDAP programs and CDAP explore queries run in the namespace. See
-       :ref:`Scheduler queue <resource-guarantees>`
-   * - ``root.directory``
-     - The pre-existing file-system directory to be used for the namespace
    * - ``hbase.namespace``
-     - The pre-existing HBase namespace to be used for the namespace
+     - The pre-existing HBase namespace to be used for the namespace (CDAP Distributed mode only)
    * - ``hive.database``
      - The pre-existing Hive database to be used for the namespace
-   * - ``principal``
-     - The Kerberos principal for the namespace
    * - ``keytabURI``
      - The URI of the Kerberos keytab file for the namespace
+   * - ``principal``
+     - The Kerberos principal for the namespace
+   * - ``root.directory``
+     - The pre-existing file-system directory to be used for the namespace
+   * - ``scheduler.queue.name``
```

Comment:
I put all of these in alphabetical order. Generally, that's what I've ended up doing with property lists, as trying to make an order out of them is impossible to maintain. We do order some of the categories.